src/mono/wasm/debugger/DebuggerTestSuite/AsyncTests.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.WebAssembly.Diagnostics;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace DebuggerTests
+{
+    public class AsyncTests : DebuggerTestBase
+    {
+
+        // FIXME: method with multiple async blocks - so that we have two separate classes for that method!
+        // FIXME: nested blocks
+        // FIXME: Confirm the actual bp location
+        // FIXME: check object properties..
+
+        //FIXME: function name
+        [Fact] // Same as Instance case
+        public async Task AsyncLocalsInContinueWithStaticBlock() => await CheckInspectLocalsAtBreakpointSite(
+             "DebuggerTests.AsyncTests.ContinueWithTests", "ContinueWithStaticAsync", 5, "<ContinueWithStaticAsync>b__3_0",
+             "window.setTimeout(function() { invoke_static_method('[debugger-test] DebuggerTests.AsyncTests.ContinueWithTests:RunAsync'); })",
+             wait_for_event_fn: async (pause_location) =>
+             {
+                var frame_locals = await GetProperties(pause_location["callFrames"][0]["callFrameId"].Value<string>());
+                Console.WriteLine (frame_locals);
```

Comment:
nit: this, and other CWLs can be dropped, unless you need them for debugging, in which case they are fine!

Reply:
done.